### DISCUSSION TITLE
Why "outline / rich-markdown-editor" become archived?

### DISCUSSION THREAD

**Discussion by ryskin** [outline/outline#2973] [General] [answered]
I wanted to use it in my project, but if it's deprecated, it doesn't make sense. Also, it's a part of https://github.com/outline/outline, so I suppose it possible it will be archived too?

Thanks,
Aleksei

**Comment by tommoor** [comment, marked as answer]
Hey! The reasoning was explained [in this post](https://github.com/outline/rich-markdown-editor/discussions/481) which has been pinned on the repo since June.

The source code for the editor has been [integrated](https://github.com/outline/outline/pull/2930) into Outline, this repo is not being archived. I recommend using [TipTap](https://tiptap.dev/) as a similar alternative.

**Reply by ryskin** [reply]
Just tried this editor and in my opinion, yours is much better.
